src/Hintman/Webhook.hs

```diff
@@ -52,8 +58,39 @@ hintmanServer = HintmanSite
     , hintmanInstallRepoRoute = repoInstalledHook
     }
 
-issueCommentHook :: MonadIO m => RepoWebhookEvent -> ((), PullRequestEvent) -> m ()
-issueCommentHook _ ev = print $ evPullReqPayload $ snd ev
+issueCommentHook
+    :: forall env m .
+       ( MonadTokenStorage m
+       , MonadUnliftIO m
+       , WithError m
+       , WithLog env m
+       , Has AppInfo env
+       )
+    => RepoWebhookEvent
+    -> ((), PullRequestEvent)
+    -> m ()
+issueCommentHook _ ((), ev) = case evPullReqAction ev of
+    PullRequestOpenedAction -> hintmanAction
+    _                       -> pass
+  where
+    hintmanAction :: m ()
+    hintmanAction = do
+        let repo = evPullReqRepo ev
+        let prInfoOwner = Owner $ getRepoOwner $ whRepoOwner repo
+        let prInfoRepo = Repo $ whRepoName repo
+        let prInfoHead = Sha $ whPullReqTargetSha $ whPullReqHead $ evPullReqPayload ev
+        let prInfoNumber = PrNumber $ evPullReqNumber ev
+
+        fetchGitHubDiff prInfoOwner prInfoRepo prInfoNumber >>= \case
+            Left err -> log E $ "Failed to fetch PR deltas: " <> toText err
+            Right prInfoDelta -> do
+                let prInfo = PrInfo{..}
+                comments <- runHLint prInfo
+                token <- itToken <$> acquireInstallationToken prInfoOwner
```

Comment:
Should we also collapse trailing spaces here?

Reply:
We can :slightly_smiling_face: 
Let's open and discuss it under the separate issue.

Reply:
See #104  :wine_glass: